Add DATE_ROUND() function to docs

Simran-B opened the pull request:
Bind parameters are not rendered!
@dothebart they were shown when we used Gitbook, weren't they?

https://5de8275e2c2330000a7f20d3--zealous-morse-14392b.netlify.com/docs/3.6/aql/functions-date.html#date_round

Need to fix rendering, I believe there was a separate button to show the BV if > 5 lines...

TODO:
- Add/better labels above pre blocks and for the buttons (based on type), maybe not strong?
- Apply layout changes consistently (arangosh, generic?)
- Check if `trim` or other filters are used which are not actually supported by *our* liquid templating engine
- Add anchors for `.example-container`
- Mobile friendliness?

jsteemann commented on 3.6/aql/functions-date.md:
```diff
@@ -352,6 +352,36 @@ DATE_TRUNC('2017-02-03', 'month') // 2017-02-01T00:00:00.000Z
+- **date** (string\|number): a date string or timestamp
+- **amount** (number): number of *unit*s
+- **unit** (string): either of the following to specify the time unit (case-insensitive):
+  - y, year, years
```
Looking at the code, I think `y`, `year` and `years` is not supported by `DATE_ROUND`.

jsteemann commented on 3.6/aql/functions-date.md:
```diff
@@ -352,6 +352,36 @@ DATE_TRUNC('2017-02-03', 'month') // 2017-02-01T00:00:00.000Z
+- **amount** (number): number of *unit*s
+- **unit** (string): either of the following to specify the time unit (case-insensitive):
+  - y, year, years
+  - m, month, months
```
Looking at the code, I think `m`, `month` and `months` is not supported by `DATE_ROUND`.

Simran-B commented:
- improved UX of aql/arangosh inline examples
- examples coming from DocuBlocks not touched! (uses inline jquery)
- disabled (invisible) anchorjs link however (arangodb/arangodb@ee70dbccc33fde87ad91ac8baaffa82b8ab674df)
- overall design needs further improvement, e.g. in case of multiple examples it is hard to see what is a query and what is a result - use a single rounded gray box, and make the background of the query, bind vars and result white?
- fixed anchor link behavior (jump to target on first click)
- added back anchor links for examples
